packages/poseidon-proof/src/to-bigint.ts

```diff
@@ -0,0 +1,20 @@
+import type { BigNumberish } from "ethers"
+import { encodeBytes32String } from "ethers/abi"
+import { toBigInt as _toBigInt } from "ethers/utils"
+
+/**
+ * Converts a bignumberish or a text to a bigint.
+ * @param value The value to be converted to bigint.
+ * @return The value converted to bigint.
+ */
+export default function toBigInt(value: BigNumberish | Uint8Array | string): bigint {
```

Comment:
Q: can be a generic utils?

Reply:
It could be moved to utils, without `ethers` though. We'd need to replace those methods with something we already have in the utils package I guess.

Reply:
makes sense - probably worth opening another issue!